package n1exercici3;

public class PruebaException {

    public static void lanzarException() {
        int[] array = new int[5];
        int index = 6;
        int valor = array[index];
    }
}
